### ISSUE TITLE
[BUG] Accessibility - Web App - There is no focus indication for keyboard navigation

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5045]
**Describe the bug**
Upon loading https://api-dev.rokwire.illinois.edu/illinois/ in Chrome Browser, I began tabbing to navigate. There was no visual focus until after the 14th Tab press. Please see video included. This renders the app inaccessible by keyboard.

**To Reproduce**
1. Go to '...https://api-dev.rokwire.illinois.edu/illinois/' in Chrome browser
2. Navigate by keyboard, pressing Tab

**Expected behavior**
With each Tab press the user can see focus shifting to next navigable element

**Actual behavior**
It takes 14 Tab presses to reach anything with a visual focus indicator.

**Screenshots**
See video. 

https://github.com/user-attachments/assets/85056b75-4c6b-44b1-9cbe-34793f167eea

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome 
 - Version 135.0.7049.85 

 - 
**Additional context**

**Comment by acovher2** [comment]
The onboarding screens are also affected by this issue.

**Comment by acovher2** [comment]
@TodorBachvarov This ticket is a bit of an EPIC. I'm not clear if the elements are not receiving focus (with no visual indicator) or if they just are not keyboard navigable to begin with. I will happily make a "Web Dev not navigable by keyboard" ticket if you'd like. Please advise. 

Thanks

**Comment by acovher2** [comment]
only thing heard by screen reader user is "blank" The user cannot tab to say yes or no to the update message. 8.0.8 version Chrome with NVDA screen reader. 

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/a5a86434-1a4a-46bd-b2cc-435dcfe4e673" />
